scripts/Pairings.js

```diff
@@ -33,3 +33,20 @@ export const getLocationServiceMatches = (locationObject, locationServicesArray,
     // Return the matched services
     return matchedServices
 }
+
+
+export const getLocationsPerService = (serviceObject, locationServicesArray, locationsArray) => {
```

Comment:
Could we have the first parameter accept an integer instead of an object?  Needing an object to be passed in creates another for..of loop in the click event function.  In the click event, I have a dataset on each item clicked that I can pass parseInt(itemClicked.dataset.id) into the function to simplify things a bit.